samples/dnn/colorization.cpp

```diff
@@ -1,128 +1,102 @@
-    -90., -80., -70., -60., -50., -40., -30., -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., 80., -110., -100., -90., -80., -70.,
-    -60., -50., -40., -30., -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., -110., -100., -90., -80., -70., -60., -50., -40., -30.,
-    -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., -90., -80., -70., -60., -50., -40., -30., -20., -10., 0.
-};
-
-int main(int argc, char **argv)
-{
+int main(int argc, char** argv) {
     const string about =
         "This sample demonstrates recoloring grayscale images with dnn.\n"
         "This program is based on:\n"
         "  http://richzhang.github.io/colorization\n"
         "  https://github.com/richzhang/colorization\n"
-        "Download caffemodel and prototxt files:\n"
-        "  http://eecs.berkeley.edu/~rich.zhang/projects/2016_colorization/files/demo_v2/colorization_release_v2.caffemodel\n"
-        "  https://raw.githubusercontent.com/richzhang/colorization/caffe/models/colorization_deploy_v2.prototxt\n";
+        "To download the onnx model:\n"
+        " https://storage.googleapis.com/ailia-models/colorization/colorizer.onnx\n";
+
     const string keys =
-        "{ h help |                                    | print this help message }"
-        "{ proto  | colorization_deploy_v2.prototxt    | model configuration }"
-        "{ model  | colorization_release_v2.caffemodel | model weights }"
-        "{ image  | space_shuttle.jpg                  | path to image file }"
-        "{ opencl |                                    | enable OpenCL }";
-    CommandLineParser parser(argc, argv, keys);
+        "{ help h          |     | Print help message. }"
+        "{ input i         | ansel_adams3.jpg | Path to the input image }"
+        "{ onnx_model_path |     | Path to the ONNX model. Required. }"
+        "{ opencl          |     | enable OpenCL }";
+
+    CommandLineParser parser(argc, argv,keys);
     parser.about(about);
-    if (parser.has("help"))
-    {
+
+    if (parser.has("help")) {
         parser.printMessage();
         return 0;
     }
-    string modelTxt = samples::findFile(parser.get<string>("proto"));
-    string modelBin = samples::findFile(parser.get<string>("model"));
-    string imageFile = samples::findFile(parser.get<string>("image"));
-    bool useOpenCL = parser.has("opencl");
-    if (!parser.check())
-    {
-        parser.printErrors();
-        return 1;
+
+    string inputImagePath = parser.get<string>("input");
+    string onnxModelPath = parser.get<string>("onnx_model_path");
+
+    if (onnxModelPath.empty()) {
+        cerr << "The path to the ONNX model is required!" << endl;
+        return -1;
     }
 
-    Mat img = imread(imageFile);
-    if (img.empty())
-    {
-        cout << "Can't read image from file: " << imageFile << endl;
-        return 2;
+    Mat img = imread(samples::findFile(inputImagePath));
```

Comment:
You can use `IMREAD_GRAYSCALE` option for `imread` and re-use lena.jpg image. No need to add the new file.

Reply:
why not use baboon.jpg, messi.jpg building.jpg?